EmployeeLib/LSPModel/ISuperVisor.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace EmployeeLib.LSPModel
+{
+    public interface ISuperVisor
+    {
+        public IEmployee SuperVisor { get; set; } 
+        // Manager renamed to supervisor because a class can't have a memeber with the same name
+        // class Manager { public Manager }  X
+
+        public void AssignManager(IEmployee manager);
```

Comment:
assign manager should not take type of employee, it should be a manager. similar to what we've done before on OCP